monitoring/.env.example

```
@@ -0,0 +1,44 @@
+
+# ─── Backend scrape target ──────────────────────────────────────────────────
+# host.docker.internal resolves to the Docker host (macOS native;
+# Linux via the host-gateway alias declared in docker-compose.yml).
+# In Coolify, set this to the backend service DNS name, e.g. steam5-backend:8081.
+STEAM5_METRICS_TARGET=host.docker.internal:8081
+STEAM5_METRICS_SCHEME=http
+STEAM5_METRICS_PATH=/actuator/prometheus
+# Basic-auth credentials Prometheus uses to scrape the backend.
+# Backend dev defaults are metrics / metrics. Override in production.
+STEAM5_METRICS_USERNAME=metrics
+STEAM5_METRICS_PASSWORD=metrics
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The STEAM5_METRICS_PASSWORD key should go before the STEAM5_METRICS_PATH key

(UnorderedKey)